### ISSUE TITLE
Annimations left/right without gap (infinite scrolling)

### ISSUE THREAD

**Issue by ditschi** [fossasia/badge-magic-android#887]
Currently when selecting the animation for scrolling left or right there is a gap where no text or Clipart is visible. It would be nice to have an option to immediate ly repeat the text after  ti ended to get a continuous loop without a gap. A current workaround is to add the same text several times so the gap is not visible all the time but only after the longer annimation was finished completely

**Comment by kienvo** [comment]
This would be accomplished easily by making a configurable setting to turn on/off infinite scrolling for the already-existed left and right animation in the firmware.

**Comment by hpdang** [comment]
@kienvo do you mean infinite scrolling is already implemented in the firmware?

**Comment by kienvo** [comment]
No, but I can implement it.

**Comment by nope3472** [comment]
@kienvo we just wanted to confirm that does infinite scrolling means multiple instance of the text on the badge at the same time with out waiting for the previous text to move out of the badge?

**Comment by kienvo** [comment]
> we just wanted to confirm that does infinite scrolling means multiple instance of the text on the badge at the same time with out waiting for the previous text to move out of the badge?

That's correct.

**Comment by kienvo** [comment]
The problem here is how the app handles this feature when using with the OEM firmware as it doesn't have this feature.

**Comment by hpdang** [comment]
Why should we support the OEM firmware? We want to implement this feature on our firmware. As discussed in our last meeting, @nope3472 you don't need to worry about this issue now. We can re-visit it once the infinite scrolling is implemented on our firmware.

**Comment by kienvo** [comment]
How does the app know which firmware it connected to. That's is the problem.
